spec/core/array/initialize_spec.rb

```diff
@@ -0,0 +1,158 @@
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+
+describe "Array#initialize" do
+  before :each do
+    ScratchPad.clear
+  end
+
+  #FIXME add back once we support have_private_instance_method
+  xit "is private" do
+    Array.should have_private_instance_method("initialize")
```

Comment:
I think we can still make `Array#initialize` private by using the new `:visibility` keyword on the bindings :^) This would not make the spec pass, but as soon as we support #private_instance_method? it will just pass

Reply:
Fair point, I am in general not a fan of writing code without tests or that cannot be tested, but in this case it shouldn't hurt :^) Thanks!

Reply:
Yeah I thought about implementing `#private_instance_method?` for the `Hash#initialize` spec, but I decided to hack around it in the spec (https://github.com/seven1m/natalie/blob/master/spec/core/hash/initialize_spec.rb#L7).

Reply:
Fixed it, but this really makes me feel like implementing this missing bit in module, I will probably create an issue to make Module spec compliant :)